src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java

```diff
@@ -285,6 +332,28 @@ private void transferRoutes(){
                 });
     }
 
+    private ResponseEntity<String> fetchApibyWorkflowKey(HttpEntity<String> entity, long workflowInstanceKey) {
+        return restTemplate.exchange(operationsUrl + "/transfer/" +
```

Comment:
can we use camel instead of rest template?  rest template is going to be deprecated.